src/App/Fossa/Analyze/Log4jReport.hs

```diff
@@ -236,7 +236,7 @@ data Log4jVulnerableReportItem = Log4jVulnerableReportItem
   deriving (Show, Eq, Ord)
 
 data VulnerableDependencyOrigin
-  = VulnerableDependencyManifestFiles [SomeBase File]
+  = VulnerableDependencyManifestFiles [Text]
```

Comment:
Consider using `SomePath` instead, which loosens the restrictions as much as possible while still indicating that we want this to mean a "path" of some kind.

Reference: https://github.com/fossas/fossa-cli/blob/2da715e852e15375094e998fc27f161aad757a52/src/Path/Extra.hs#L19-L24

Reply:
Thanks, I have made this change.